wpilibc/src/main/native/include/frc/ADIS16448_IMU.h

```diff
@@ -419,6 +419,7 @@ class ADIS16448_IMU : public wpi::Sendable,
   std::atomic<bool> m_first_run = true;
   std::atomic<bool> m_thread_idle = false;
   std::atomic<bool> m_start_up_mode = true;
+  std::atomic<bool> m_needs_flash = false;
```

Comment:
Any reason why this can't be a local variable of the constructor? I don't see anyplace outside of the constructor that uses it.

Reply:
No real reason other than wanting to preallocate the memory. It's a Boolean, so 🤷‍♂️

Reply:
It's only used in the constructor from a single thread, so it can be a local `bool` instead of a `std::atomic<bool>` member variable.  It'll likely be optimized to a register read/write and not allocate any stack memory.

Reply:
Done.